package.json

```diff
@@ -28,7 +28,7 @@
     "react-infinite-scroller": "^1.2.4",
     "react-lazy-load-image-component": "^1.5.0",
     "react-redux": "^7.1.0",
-    "react-router": "^4.3.1",
+    "react-router": "^5.0.0",
```

Comment:
Is this line really necessary? I don't recall us having it before.

Reply:
It was a missing dependency. If I didn't include, it would pop up every time we ran yarn.

Reply:
Hmm. Likely `npm` popped it in without being explicitly requested?

Reply:
<img width="837" alt="Screen Shot 2020-10-27 at 8 33 39 PM" src="https://user-images.githubusercontent.com/5706163/97387671-b4f2c400-1893-11eb-9351-b2f142c9ec4c.png">
`npm` doesn't warn on install, but those packages have been missing.